Helper/ListOptions.php

```diff
@@ -22,14 +22,14 @@ public function toOptionArray()
         // field in Klaviyo\Reclaim\Block\System\Config\Form\Field\Newsletter, so we pass
         // it over in the options array.
 
-        if (!$this->data_helper->getPrivateApiKey()) {
+        if (!$this->_dataHelper->getPrivateApiKey()) {
             return [[
                 'label' => 'To sync newsletter subscribers to Klaviyo, first save a <strong>Private Klaviyo API Key</strong> on the "General" tab.',
                 'value' => 0
```

Comment:
we're using 'label' and 'value' strings all over this sucker. could constant those bad boys